lib/fizzy/capi.cpp

```diff
@@ -45,14 +45,16 @@ inline fizzy::ValType unwrap(FizzyValueType value_type) noexcept
 
 inline FizzyFunctionType wrap(const fizzy::FuncType& type) noexcept
 {
-    return {(type.outputs.empty() ? FizzyValueTypeVoid : wrap(type.outputs[0])),
+    return {(type.outputs.empty() ? static_cast<FizzyValueType>(FizzyValueTypeVoid) :
```

Comment:
```suggestion
    return {(type.outputs.empty() ? FizzyValueType{FizzyValueTypeVoid} :
```
Nicer conversion may be possible here. https://godbolt.org/z/o9Y4jK